mava/wrappers/jumanji.py

```diff
@@ -217,3 +219,101 @@ def observation_spec(self) -> specs.Spec[Union[Observation, ObservationGlobalSta
             )
 
         return spec
+
+
+class CleanerWrapper(MultiAgentWrapper):
+    """Multi-agent wrapper for the Cleaner environment.
+
+    Do not use the AgentID wrapper with this env, it has implicit agent IDs.
```

Comment:
Previously the comment was here since `add_agent_id ` is set in the system config, not the env config. But with the global state merge environments have the `implicit_agent_id` flag, so yes the comment is no longer necessary.